packages/@okta/vuepress-theme-prose/global-components/SignUp.vue

```diff
@@ -255,9 +299,27 @@ export default {
       this.validationService.checkFormInput("country");
       this.validationService.checkEmailInput("email");
       this.validationService.checkFormInput("state");
+      this.validationService.checkFormCheckboxInput("consentAgree");
 
       if (this.validationService.isValidForm()) {
-       // make api call
+        // make api call
+      }
+    },
+
+    showConsentSection(country) {
+      // Show consent section while selecting country.
+      this.displayConsent = true;
+      // Hide consent opt checkbox.
+      this.displayAgree = false;
+      this.form.consentAgree.hidden = true;
+      this.validationService.resetFormField("consentAgree", {
+        reset: true,
+        value: false
+      });
+
+      if (GDPR_COUNTRIES.indexOf(country) !== -1) {
```

Comment:
Can we use `.contains`? It's more appropriate.

Reply:
Never mind, it looks like vuepress version we use only has `core-js` 2.x which at the time only had `.includes` as a Stage 4 proposal so it's probably not on by default. We'll just keep `indexOf`.